Food/sops.lua

```diff
@@ -0,0 +1,53 @@
+local name = "sops"
+local version = "3.7.1"
+local release = "v" .. version
+local org = "mozilla"
+local repo = "sops"
+local url = "https://github.com/" .. org .. "/" .. repo
+
+food = {
+    name = name,
+    description = "sops is an editor of encrypted files that supports YAML, JSON, ENV, INI and BINARY formats and encrypts with AWS KMS, GCP KMS, Azure Key Vault, age, and PGP",
+    homepage = url,
+    version = version,
+    packages = {
+        {
+            os = "darwin",
+            arch = "amd64",
+            url = url .. "/releases/download/" .. release .. "/" .. name .. "-" .. release .. ".darwin",
+            sha256 = "43d2f9c63921a57bf607268a05d480cc309e9979bb81269248dd117e5efac133",
+            resources = {
+                {
+                    path = name,
```

Comment:
```suggestion
                    path = name .. "-" .. release .. ".darwin",
```
- `name` represents the asset here, which is actually `name .. "-" .. release .. ".darwin"`
- `installpath = "bin/" .. name` will change `sops-v0.0.0.darwin` to `sops`